docs/usage.md

```diff
@@ -0,0 +1,63 @@
+This chapter describes how to enhance your web application by server-side 
+UI-contributions.  
+
+The chartlets backend implementation is provided by the Python package `chartlets`.
+It makes it easy to implement the chartlet endpoints in your preferred
+webserver framework, such as Flask, FastAPI, or Tornado. 
+
+### 1. Implement the possible contributions
+
+Implement the application-specific contributions that users 
+can add to their extensions.
+
+As an example, see [`panel.py` of the demo](chartlets.py/chartlets/demo/contribs/panel.py):
```

Comment:
This applies to all the links below as well, they all return 404. Are they yet to be added in issue https://github.com/bcdev/chartlets/issues/34?

Reply:
I can onlöy link to them once we have deployed the docs on GH pages.